docs/community/documentation-style-guide.rst

```diff
@@ -0,0 +1,377 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2017-2019 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+.. _documentation-style-guide:
+
+=========================
+Documentation Style Guide
+=========================
+
+
+General style and tone
+----------------------
+Starting to write documentation can seem like a daunting task, but remember
+that not only other members of the community but you yourself will use this
+valuable resource.
+
+This guide is meant to help by providing you with quick pointers and
+inform you as to what you should cover.
```

Comment:
I'd make this a bit clearer. ```as to what you should cover when writing documentation```

Reply:
Also, maybe this introductory part shouldn't be in the General style and tone section?